docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1099 @@
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the recognized commands selection callback invoked when the voice control manager select specific voice commands in all recognized result:
+
+    When utterance is recognized, all the recognized result can include more than two voice commands if these commands have the same command string that is registered by each voice control clients.
+
+    In this case, voice control manager can select specific voice commands from all recognized result to send voice control client using `SetRecognizedCommandsSelectionDelegate`.
+
+    You can select a valid command result from the recognized command selection.
+
+    ```csharp
+    /// callback
+    IEnumerable<VoiceCommand> recognizedCommandsSelection(IEnumerable<VoiceCommand> commands, string recognizedText, string message)
+    {
+        /// Your code
+        return null;    /// release commands.
+
+        /// Select valid results
+    }
+
+    void SetUnsetRecognizedCommandsSelectionCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate(recognizedCommandsSelection);
+
+            /// Unset the callback
+            VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate(null);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set a callback function to be called when engine sends audio formats necessary for playing TTS feedback:
+    ```csharp
+    /// callback
+    void feedbackAudioFormatChanged(object sender, FeedbackAudioFormatEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetFeedbackAudioFormatChangedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.FeedbackAudioFormatChanged += feedbackAudioFormatChanged;
+
+            /// Unset the callback
+            VoiceControlManagerClient.FeedbackAudioFormatChanged -= feedbackAudioFormatChanged;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set a callback function to be called when engine sends audio streaming for TTS feedback:
+    ```csharp
+    /// callback
+    void feedbackStreaming(object sender, FeedbackStreamingEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetFeedbackStreamingCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.FeedbackStreaming += feedbackStreaming;
+
+            /// Unset the callback
+            VoiceControlManagerClient.FeedbackStreaming -= feedbackStreaming;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set TTS streaming callback function to be called when the VC-Client sends audio streaming for TTS feedback:
```

Comment:
@surbhig12 for "VC-Client", how about "VC-Client(voice control client)"?

Reply:
@mijin85cho, as per our style guide, I suggest changing to:

Set TTS streaming callback function to be called when the voice control client (VC-Client) sends audio streaming for TTS feedback:

Reply:
@mijin85cho, i agree to your suggestion https://github.com/Samsung/tizen-docs/pull/1048#discussion_r457971995

Request @sungraejo, please incorporate the comment.

Reply:
Thanks for the rewritting @surbhig12

Reply:
@surbhig12 @mijin85cho 
Done. Thank you your suggestion.